### DISCUSSION TITLE
Adding assets to build process using vite

### DISCUSSION THREAD

**Discussion by jaspercjc** [statamic/cms#6662] [Troubleshooting]
I wanted to create a custom fieldtype, following the [docs](https://statamic.dev/extending/fieldtypes) but got blocked on [this](https://statamic.dev/extending/control-panel#adding-assets-to-your-build-process) part.

How do I implement or convert 
`mix.js('resources/js/cp.js', 'public/vendor/app/js')
   .sass('resources/sass/cp.scss', 'public/vendor/app/css');`
to vite equivalent?

**Comment by philliphartin** [comment]
Got bogged here myself. @jaspercjc did you ever figure it out?

**Comment by richdynamix** [comment]
I am also in the same boat. Can't figure out the vite config for the same actions. I think the docs need updating.

**Comment by jasonvarga** [comment]
Vite looks like a bit of a different beast at the moment. I'd say we don't support it just yet, and you should continue to use webpack.

There is now a feature request open on the ideas repo where you can follow along. https://github.com/statamic/ideas/issues/876
